recipes/cleangeostreamr/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "cleangeostreamr" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name }}
```

Most conda R packages have a prefix `r-`

```suggestion
  name: r-{{ name }}
```